### PR TITLE
Added Header and Footer components

### PR DESCRIPTION
**Pull request by annedeg**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -2,10 +2,14 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { HeaderComponent } from './general-components/header/header.component';
```
Comment (Nickerty) [inline src/app/app.module.ts:5]: Is importing header & footer components 100% necessary in app.module?

Comment (annedeg) [reply src/app/app.module.ts]: Just updated this. Could you check?

Comment (Nickerty) [reply src/app/app.module.ts]: Seems much better!

File: src/app/app.module.ts [hunk 2]
```diff
@@ -2,13 +2,15 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { GeneralComponentsModule } from './general-components/general-components.module';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
```
Comment (Nickerty) [inline src/app/app.module.ts:9]: UNACCPTABLE

File: src/app/general-components/footer/footer.component.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="navigation">
+    <div class="container">
+        <span>
+            Social Distancing Network | Copyright {{apple}}
```
Comment (Nickerty) [inline src/app/general-components/footer/footer.component.html:4]: UNACCEPTABLE

File: src/app/general-components/footer/footer.component.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
+})
+export class FooterComponent implements OnInit {
+
+  constructor() { }
+  //First item of array is button text
+  //Second item of array is id
+  navigation = [["Home", 0], ["Contact",1], ["Overig", 2]]
+  apple = new Date().getFullYear()
```
Comment (Nickerty) [inline src/app/general-components/footer/footer.component.ts:14]: THIS CODE IS

**UNACCEPTABLEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE**

File: src/app/general-components/header/header.component.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+
+  loginName = "Anne"
```
Comment (Nickerty) [inline src/app/general-components/header/header.component.ts:10]: image

**Pull request merged**